LifeCycleAssessment_Engine/LifeCycleAssessment_Engine.csproj

```diff
@@ -128,6 +128,9 @@
     <Compile Include="Compute\EvaluateEnvironmentalProductDeclarationByMass.cs" />
     <Compile Include="Compute\EvaluateReferenceValue.cs" />
     <Compile Include="Objects\HelperMethods.cs" />
+    <Compile Include="Query\ElementEmbodiedCarbon.cs" />
+    <Compile Include="Query\EvaluateElement.cs" />
+    <Compile Include="Query\EvaluateMaterialTakeoff.cs" />
```

Comment:
Would also add L131 as well

Reply:
Following discussion with @michaelhoehn I am also of the opinion that these sound like they should be `Compute` methods rather than `Query` methods - owing to `Evaluation` sounds like `Calculation` which sounds like `Computation` rather than asking a question of an object (the purpose of `Query` methods).